test/aws/testHandler.ts

```diff
@@ -58,8 +58,8 @@ describe('AWS Handler', () => {
     const result = await (<any> promisify(awsHandler(handlerStub, 'APP')))({}, {});
     result.should.be.eql('HELLO');
 
-    bugsnagFactory.initialize.should.have.been.calledOnce();
-    AWSConfigStub.should.be.calledOnce();
+    bugsnagFactory.initialize.calledOnce.should.be.True();
+    AWSConfigStub.calledOnce.should.be.True();
```

Comment:
This change seems fine to me, but what is the difference between the two?

Reply:
ignore this comment, just read your message for the pr above

Reply:
yeah, was getting errors like this
![image](https://user-images.githubusercontent.com/54287973/112700109-a280b480-8e4a-11eb-8133-34980f97d28e.png)